cmd/sink/logs/logs.go

```diff
@@ -0,0 +1,93 @@
+// Copyright KubeArchive Authors
+// SPDX-License-Identifier: Apache-2.0
+
+package logs
+
+import (
+	"context"
+	"fmt"
+	"strings"
+
+	ocel "github.com/kubearchive/kubearchive/pkg/cel"
+	"github.com/kubearchive/kubearchive/pkg/log_urls"
+	corev1 "k8s.io/api/core/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/apis/meta/v1/unstructured"
+	"k8s.io/client-go/kubernetes"
+)
+
+const celPrefix = "cel:"
+
+var (
+	configCmRef = corev1.ObjectReference{
+		Kind:      "ConfigMap",
+		Name:      "kubearchive-config",
+		Namespace: "kubearchive",
+	}
+	loggingCmRef = corev1.ObjectReference{
+		Kind:      "ConfigMap",
+		Namespace: "kubearchive",
+	}
+	loggingCmSelector = corev1.ConfigMapKeySelector{
+		Key: "logging-configmap-name",
+	}
+)
+
+func getKubeArchiveLoggingCm(ctx context.Context, kubeClient kubernetes.Interface) *corev1.ConfigMap {
+	emptyCm := &corev1.ConfigMap{Data: map[string]string{}}
+	kaConfigCm, err := kubeClient.CoreV1().ConfigMaps(configCmRef.Namespace).Get(
+		ctx,
+		configCmRef.Name,
+		metav1.GetOptions{},
+	)
+	if err != nil {
+		return emptyCm
```

Comment:
We need to log the error to show whats happening, maybe something like:
```suggestion
        slog.Warn("error while getting configmap", "configmap", fmt.Sprintf("%s/%s", configCmRef.Namespace, configCmRef.Name), "error", err.Error())
		return emptyCm
```